<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QLabel" name="label_Registration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>242</width>
     <height>47</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>71</y>
     <width>601</width>
     <height>358</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_email">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_ReTypePassword">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Re-type Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_ReTypePassword">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_lastname">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lastname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_lastname">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_firstname">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Firstname</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_firstname">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_male">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_female">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_other">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_dob">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DOB</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="displayFormat">
       <string>dd-MM-yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_phone">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_phone">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;
</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_nhi">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NHI</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_nhi">
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0.5px solid;
border-radius: 15px;
height: 30px;
padding-left: 10px;
padding-right: 10px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_ReTypePassword</tabstop>
  <tabstop>lineEdit_lastname</tabstop>
  <tabstop>lineEdit_firstname</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
